update-stemcell.sh

```diff
@@ -11,7 +11,13 @@
 
 set -o errexit -o nounset
 
-REPO="${REPO:-../../github.com/SUSE/scf}"
+if [ "${USER}" != "jan" ]; then
+    REPO="${REPO:-../../github.com/SUSE/scf}"
```

Comment:
The `REPO` part we should merge, yes (for everybody, not just you); the `helm` part… shouldn't that just be `helm` for everybody, not just you? It what's the old code did…

Reply:
Oops, I thought I had set it to `helm-2.11` for myself, and left it at `helm` for everybody else.

Will double-check why I set it to `helm-2.11` to begin with.

Reply:
Yeah, our Vagrant box uses tiller 2.11, so you can't talk to it using a later client:

```
Error: incompatible versions client[v2.13.1] server[v2.11.0]
```

Reply:
> Oops, I thought I had set it to `helm-2.11` for myself, and left it at `helm` for everybody else.

That's indeed what I had, and I will restore setting `HELM` to custom value for me, but make `REPO` the same for everyone.